specificatie/verblijfplaats.yaml

```diff
@@ -300,6 +303,8 @@ components:
               type: boolean
```

Comment:
gemeenteVanInschrijving is geschrapt uit VerblijfplaatsBuitenland, maar niet uit VerblijfplaatsBuitenlandInOnderzoek? Waarom is dat?

Reply:
over het hoofd gezien. Wordt verwijderd.

Reply:
klopt! wordt aangepast